src/components/Services/index.jsx

```diff
@@ -1,23 +1,45 @@
 import ServiceCard from "./ServiceCard";
+import ServicesInfo from "./ServicesInfo";
 import data from "./data";
+import { useState } from "react";
 
 const Services = () => {
+  const [isModalOpen, setModalOpen] = useState(false);
+  const [selectedService, setSelectedService] = useState(null);
+  const [selectedServiceInfo, setSelectedServiceInfo] = useState(null);
```

Comment:
Why not merge these 2 states into a single one? They are always set and read together, no need for different pieces of state.